plugins/kubectl_ai.yaml

```diff
@@ -0,0 +1,13 @@
+plugin:
```

Comment:
For those that don't read the main readme ;(
We should have a brief header with the project url and how to install it